setup.py

```diff
@@ -9,11 +10,15 @@
     "plone.testing",
 ]
 
+long_description = (
+    f"{Path('README.rst').read_text()}\n{Path('CHANGES.rst').read_text()}"
```

Comment:
I always do this:

```
with open("README.rst") as myfile:
    readme = myfile.read()
```

But `pathlib.Path.read_text` looks much simpler.  We should be using `pathlib` much more in Plone!